### ISSUE TITLE
NVDA screen reader does not read anchor href links(role="link") as links in Windows Outlook app

### ISSUE THREAD

**Issue by pavig30** [nvaccess/nvda#13949]
### Steps to reproduce:
Create a simple plain barebones table structure like below. Save it to a file with .html extension. Send email to Windows Outlook App. Open the NVDA screen reader and tab to the email body. NVDA screen reader reads the first link in the email body as a link and remaining as text. If you navigate back to the link which was read as a link earlier, it will be read as a text the second time. 

``` HTML
<table role="presentation" cellspacing="0" cellpadding="0" border="0">
    <tbody>
        <tr>
            <td>
                <p>These are created for testing purposes</p>
                <a href="https://www.google.com/">Link A</a><br />
                <a href="https://www.google.com/">Link B</a><br />
                <a href="https://www.google.com/">Link C</a>
            </td>
        </tr>
    </tbody>
</table>
```

### Expected behavior:

NVDA screen reader should read all anchor href links as a links.
link Link A
link Link B
link Link C
#### NVDA installed/portable/running from source:
NVDA installed
#### NVDA version:
nvda_2022.1
#### Windows version:
Windows 10

#### Name and version of other software in use when reproducing the issue:
NA
#### Other information about your system:
NA
#### Does the issue still occur after restarting your computer?
Yes
#### Have you tried any other versions of NVDA? If so, please report their behaviors.
No
#### If NVDA add-ons are disabled, is your problem still occurring?
Yes
#### Does the issue still occur after you run the COM Registration Fixing Tool in NVDA's tools menu?
Yes

**Comment by Adriani90** [comment]
@pavig30 I cannot reproduce this issue with NVDA 2022.4 and Outlook 365 Version 2302 Build 16.0.16130.20186). Please report which Outlook version are you using?

**Comment by seanbudd** [comment]
Closing this as abandoned as the earlier question was never answered.
If you can answer the outlying questions and can still reproduce this let us know
